src/test/java/org/opentripplanner/middleware/testutils/ApiTestUtils.java

```diff
@@ -90,7 +90,7 @@
      * Attempt to get Auth0 token from Auth0 tenant for the given username (email) and scope. If the response from Auth0
      * is ok, extract the access token from the token holder response and return to caller.
      */
-    private static String getTestAuth0AccessToken(String username, String scope) {
+    private static String getTestAuth0AccessToken(String username, String scope) throws Exception {
```

Comment:
Do we want this to throw an exception or should we simply swallow the exception generated by:
```
TokenHolder token = JsonUtils.getPOJOFromJSON(response.responseBody, TokenHolder.class);
```
I suppose the effect of throwing the exception would be that the test would automatically fail? That seems like acceptable behavior to me, so perhaps we keep the exception. I will say though that the appearance of a generic exception on all of these methods rather than `JsonProcessingException` had me scratching my head for a bit.

Reply:
Yeah, the tests would just fail. I started putting up a bunch of JsonProcessingExceptions, but sometimes those got appended to other exceptions that it just seemed like adding a generic Exception would just catch any error and fail the test. Of course, the exceptions should be handled differently in non-test code, but this seemed appropriate to me to liberally add generic Exceptions being thrown from various test methods.